src/actions/mintEditionFromMaster.ts

```diff
@@ -46,6 +46,7 @@ export const mintEditionFromMaster = async (
     TOKEN_PROGRAM_ID,
     masterEditionMint,
     wallet.publicKey,
+    true,
```

Comment:
As mentioned by @kurpav, this change probably needs better argumentation in form of a source-code comment, due to its' above average nuance. The intent is quite cryptic here and is guaranteed to be lost with time (without a substantial research of intent by readers)

If I understand correctly, the benefit of setting `allowOwnerOffCurve` to `true` is that we allow to mint tokens owned by Program accounts? (whose ATA *must* fall outside of ed25519 curve, so that there isn't any private key for it). If so, it'd be useful to comment here that this is the intent.

Is there any other intent that I missed?